### PR TITLE
Allow rolling back player data

### PR DESCRIPTION
**Pull request by jasonw4331**

### CODE REVIEW

File: src/jasonwynn10/GitRollbacks/GitRepository.php [hunk 1]
```diff
@@ -699,6 +699,7 @@ protected function run($cmd) : self
 			$args = func_get_args();
 			$cmd = self::processCommand($args);
 			exec($cmd . ' 2>&1', $output, $ret);
+			var_dump($output);
```
Comment (jasonw4331) [inline src/jasonwynn10/GitRollbacks/GitRepository.php]: needs removed

File: plugin.yml [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
 main: jasonwynn10\GitRollbacks\Main
-version: 0.1.0
+version: 0.2.0
 api: 3.0.0
 description: ""
 author: "jasonwynn10"
-load: STARTUP
\ No newline at end of file
+load: STARTUP
+os:
```
Comment (inxomnyaa) [inline plugin.yml:8]: Just here for consistency? Having all 3 makes this obsolete afaik

Comment (jasonwynn10) [reply plugin.yml]: There are more than 3 operating systems... these 3 are the ones supported.

Comment (inxomnyaa) [reply plugin.yml:8]: Hm, but afair pocketmine only checks for win, mac und linux.. Sorry if i am mistaken ^^"

Comment (jasonw4331) [reply plugin.yml:8]: Pocketmine checks for Windows, Linux, Android, mac, iOS, and something else I can’t remember. Look at `Utils::getOS();` to see what i mean

**Pull request merged**